Sources/Gui/SDLRunner.cpp

```diff
@@ -518,7 +522,8 @@ namespace spades {
 
 #ifndef __sun
 					if (rtype == RendererType::GL) {
-						if (r_vsync != 0 && SDL_GL_SetSwapInterval(r_vsync) != 0) {
+						int vsync = r_vsync;
+						if (vsync != 0 && SDL_GL_SetSwapInterval(vsync) != 0) {
```

Comment:
Nice change but really belongs to a separate PR.

Reply:
Not a nice change but it s in fact a fix, you asked earlier if I tried out the default with -1 and because I did I found out the value were incorrect so needed the explicit cast.

Reply:
Could you revert this change as well?